wire-ios-request-strategy/Sources/Request Strategies/Assets/Helpers/AssetDownloadRequestFactory.swift

```diff
@@ -23,21 +23,21 @@ public final class AssetDownloadRequestFactory: NSObject {
 
     public func requestToGetAsset(withKey key: String, token: String?, domain: String?, apiVersion: APIVersion) -> ZMTransportRequest? {
         let path: String
+        let domain = domain?.isEmpty == false ? domain! : BackendInfo.domain
```

Comment:
`domain?.isEmpty == false` can it be `true` if domain is `nil`? I guess no, but not completely clear to me..

Do you like this variant?
```suggestion
        let resolvedDomain: String
        if let domain, !domain.isEmpty {
            resolvedDomain = domain
        } else {
            resolvedDomain = BackendInfo.domain
        }
```